### DISCUSSION TITLE
extension discontinued?

### DISCUSSION THREAD

**Discussion by ghost** [ungoogled-software/ungoogled-chromium-android#80] [Q&A] [answered]
i came here to ask why you discontinue this variant?

did the newer version webview support extension

did the newer version has it own webview?

if yes how i can change the webview without root?

@wchen342

**Comment by wchen342** [comment, marked as answer]
> why you discontinue this variant

I explained in #78.

> did the newer version webview support extension

No.

> did the newer version has it own webview?

Yes.

> how i can change the webview without root

No you can't. Android locks webview to specific package signature.

**Comment by matuszajac** [comment]
please continue with extension version. 
the best browser with extension for nextcloud. 
dont care about cookies. 
adblocking...
